src/pineko/evolve.py

```diff
@@ -39,10 +42,10 @@ def write_operator_card_from_file(pineappl_path, default_card_path, card_path):
     pineappl_grid = pineappl.grid.Grid.read(pineappl_path)
     with open(default_card_path, "r", encoding="UTF-8") as f:
         default_card = yaml.safe_load(f)
-    return write_operator_card(pineappl_grid, default_card, card_path)
+    return write_operator_card(pineappl_grid, default_card, card_path, xif)
 
 
-def write_operator_card(pineappl_grid, default_card, card_path):
+def write_operator_card(pineappl_grid, default_card, card_path, xi):
```

Comment:
Why do you want to call it `xi`? It is always `xif`, right? So maybe would be better to just call it `xif`